pyproject.toml

```diff
@@ -1,8 +1,25 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
 [project]
-name = "auto-gpt"
-version = "0.1.0"
-description = "A GPT based ai agent"
+name = "agpt"
+version = "0.2.2"
+authors = [
+  { name="Torantulino", email="support@agpt.co" },
+]
 readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
3.10?

Reply:
You can force edit if you like, I was just trying to make the name and email right.